### PR TITLE
fix: cell height style

### PR DESCRIPTION
**Pull request by alexstaroselsky**
Update cell `style` to ensure size unit "px" is specified.

### CODE REVIEW

File: src/VirtualGrid.vue [hunk 1]
```diff
@@ -422,7 +422,7 @@ export default class VirtualGrid<P> extends Vue {
                 v-for="item in renderData.cellsToRender"
                 :key="item.id"
                 :style="{
-                    'height': item.height,
+                    'height': item.height + 'px',
```
Comment (Mikescops) [inline src/VirtualGrid.vue:425]: Nice catch, I'd suggest this format if I may:

```suggestion
                    'height': `${item.height}px`,
```

Comment (alexstaroselsky) [reply src/VirtualGrid.vue]: Updated to use string interpolation.

**Review by Mikescops — APPROVED** [review]

**Comment by Mikescops** [comment]
Thanks!

**Comment by Mikescops** [comment]
This PR is included in release v2.4.0

**Pull request merged**